sys-apps/mount-gtk/mount-gtk-1.4.4.ebuild

```
@@ -37,5 +37,5 @@ src_prepare() {
 src_configure() {
 	# acinclude.m4 is broken and environment flags override these:
 	append-cxxflags -fexceptions -frtti -fsigned-char -fno-check-new -pthread
-	econf --docdir=/usr/share/doc/${PF}
+	default
```

```suggestion
	econf
```
Probably a matter of taste, but I'd leave the explicit command in place.